client/my-sites/marketing/do-it-for-me/site-info-collection.tsx

```diff
@@ -13,6 +13,11 @@ import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopp
 import { getCurrentUserName } from 'calypso/state/current-user/selectors';
```

Comment:
Thought of leaving the existing landing page flow in tact just in case we want to have a landing page. Hoping to clean those assets!